src/components/ChartMenu/ChartMenu.tsx

```diff
@@ -115,6 +67,12 @@ export const ChartMenu = () => {
               key={chartElement.id}
               chartElement={chartElement}
               handleDropOnChart={handleDropOnChart}
+              removeElement={() => {
+                removeElement(chartElement.id);
```

Comment:
I think u dont need to put the curly braces: removeElement={ () => removeElement(chartElement.id) }